### ISSUE TITLE
CyElement onPositionChange undefined

### ISSUE THREAD

**Issue by PadreSVK** [rcarcasses/vue-cytoscape#51]
Hi I have problem with ```onPositionChange``` method when parameter ```position``` have value ```undefined```.  I dont know from where could be this undefined value, because all my edges and nodes have property ```position``` (some of them with empty array). 
I use same cytoscape instance for more "graphs" and this issue appears when I change graph (nodes, edges and config).

```vue
<cytoscape
    ref="cy"
    :preConfig="preConfig"
    :afterCreated="afterCreated"
    :config="config"
    class="viewport-y-full"
>
    <cy-element
        v-for="node in nodes"
        :key="`${node.data.id}`"
        :definition="node"
        v-on:click="graphNodeClick($event)"
    />
    <cy-element v-for="edge in edges" :key="`${edge.data.id}`" :definition="edge" />
</cytoscape>
```
It is possible to add default value ```null``` for parameter  ```position```?  ```null``` is friendly with ```JSON.parse``` 😊
https://github.com/rcarcasses/vue-cytoscape/blob/master/src/components/CyElement.ts#L96

**Comment by rcarcasses** [comment]
Hi @PadreSVK, looks fine to me, can you bump also package version in `package.json` in your pull request?

**Comment by PadreSVK** [comment]
sure 👍

**Comment by PadreSVK** [comment]
It is ok version in PR title or could be elsewhere?

**Comment by rcarcasses** [comment]
merged, update to version `1.0.8`.
